build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/operator-framework/ansible-operator:v0.17.0
+FROM quay.io/operator-framework/ansible-operator:v1.5.0
```

Comment:
wonder if this should point to latest? we do that for mig-operator

https://github.com/konveyor/mig-operator/blob/master/build/Dockerfile#L1

Reply:
Agreed, we've been actively moving to use :latest unless we know of a reason we can't.

Reply:
I was actually going to say we should probably pin mig-operator to a stable release :)

Reply:
We had an issue with pinning in past of forgetting to update and then not picking up newer versions that addressed CVEs.  The CVEs were the main driver to move us to consume :latest.  There was also an impression that ansible-operator has matured so less concern of breaking, and if we did break on a new update good to learn ASAP and catch the issue.

Reply:
Thanks, I will move to latest